bro_connector/main/management/commands/frd_sync_to_bro.py

```diff
@@ -1,13 +1,396 @@
+import os
+import time
+
 from django.core.management.base import BaseCommand
+from bro_exchange.broxml.frd.requests import FrdStartregistrationTool
+from bro_exchange.bhp.connector import (
+    validate_sourcedoc,
+    upload_sourcedocs_from_dict,
+    check_delivery_status,
+)
+from frd.models import FormationResistanceDossier, FrdSyncLog
+from datetime import datetime
+from main.settings.base import FRD_SETTINGS
+
+#############
+## Command ##
+#############
 
 
 class Command(BaseCommand):
     """
     Handles the sync between the bro-connector and the BRO.
     Runs daily as cronjob.
     """
-    help = 'Syncs the FRD data to the BRO'
+
+    help = "Syncs the FRD data to the BRO"
 
     def handle(self, *args, **kwargs):
+        self.handle_startregistrations()
+        self.handle_measurements()
+
+    def handle_startregistrations(self):
+        """
+        Handles the startregistrations for all FRD's without bro_id.
+        For each new FRD, the FrdStartregistration class is used to hande the registration
+        """
+        new_dossiers_qs = self.get_new_dossiers()
+
+        if len(new_dossiers_qs) != 0:
+            for new_dossier in new_dossiers_qs:
+                startregistration = FrdStartregistration(new_dossier)
+                startregistration.sync()
+
+    def get_new_dossiers(self):
+        return FormationResistanceDossier.objects.filter(frd_bro_id=None)
+
+    def handle_measurements(self):
+        """
+        Develop when startregistrations are done
+        """
         pass
 
+
+##############################
+## General helper functions ##
+##############################
+
+
+def update_log_obj(log_obj, field_value_dict):
+    """
+    Updates the date modified field to now, and changes the values found in the dict
+    """
```

Comment:
Een "object" heeft "values" en die staan in een "dict": iemand doet iets met van alles :-) Niet erg specifiek.

Betere namen? Wat is een "log object"? Klinkt alsos het een instantie van `FrdSyncLog` is? Dan is `frd_sync_log` als naam beter (en nog beter met een type hint).